### PR TITLE
add atomic-openshift-node sync check during upgrade

### PR DESCRIPTION
**Pull request by yocum137**

### CODE REVIEW

File: roles/openshift_node/tasks/upgrade/restart.yml [hunk 1]
```diff
@@ -57,13 +57,45 @@
   retries: 30
   until: node_upgrade_oc_csr_approve is succeeded
 
-- name: Check status of node service
-  async_status:
-    jid: "{{ node_service.ansible_job_id }}"
```
Comment (vrutkovs) [inline roles/openshift_node/tasks/upgrade/restart.yml]: I don't think its a good idea to drop `async_status` - why would we want to re-implement it with `systemctl status atomic-openshift-node` and restart it if it has failed?

**Comment by mtnbikenc** [comment]
Ref: https://bugzilla.redhat.com/show_bug.cgi?id=1705642

File: roles/openshift_node/tasks/upgrade/restart.yml (hunk 1 above)
Comment (yocum137) [reply roles/openshift_node/tasks/upgrade/restart.yml]: Is it possible to use until: with async_status:?

**Comment by sdodson** [comment]
/close
https://github.com/openshift/openshift-ansible/pull/11603 fixed this

**Pull request closed without merging**